go.mod

```diff
@@ -51,6 +54,7 @@ require (
 	github.com/klauspost/compress v1.15.9
 	github.com/koding/websocketproxy v0.0.0-20181220232114-7ed82d81a28c
 	github.com/kubevela/prism v1.4.1-0.20220613123457-94f1190f87c2
+	github.com/kubevela/workflow v0.0.0-20220828153805-1e54f0265666
```

Comment:
we must have a concrete tag

Reply:
We can use git commit tag first and change to a concrete tag before we release 1.6.